data/data/ovirt/affinity_group/main.tf

```diff
@@ -0,0 +1,9 @@
+resource "ovirt_affinity_group" "affinity_groups" {
```

Comment:
Shouldn't the `vm_list` be put in the `ignore_changes` lifecycle attribute here? After all, the PR that removes this attribute is not merged and there are some valid discussions around this.

Reply:
Well, we are not passing it so it is never changed.
The problem in the past was the dependency that we had between the bootstrap and masters that caused the affinity groups to get reevaluated on destroy but we removed that thanks to moving the lock to the affinity group function in the VM resource, so no problem now.

Reply:
Done